docs/content/docs/deployment/speculative_execution.md

```diff
@@ -0,0 +1,100 @@
+will be admitted, its output will be seen and consumed by the downstream tasks, and the remaining attempts 
+will be canceled.
+
+To achieve this, Flink uses the slow task detector to detect slow tasks. The nodes that the slow tasks
+locate in will be identified as problematic nodes and get blocked via the blocklist handler. The scheduler 
+will create new attempts for the slow tasks and deploy them on nodes that are not blocked.
+
+## Usage
+This section describes how to use speculative execution, including how to enable it, how to tuning it, and
+how to develop/improve custom sources to work with speculative execution.
+
+{{< hint warning >}}
+Note: Flink does not support speculative execution of sinks yet and will support it in follow-up releases.
+{{< /hint >}}
+
+{{< hint warning >}}
+Note: Flink does not support speculative execution of DataSet jobs because DataSet will be deprecated 
+in near future. DataStream API is now the recommended low level API to develop Flink batch jobs.
+{{< /hint >}}
+
+### Enable Speculative Execution
+To enable speculative execution, you need to set the following configuration options:
+- `jobmanager.scheduler: AdaptiveBatch`
+    - Because only [Adaptive Batch Scheduler]({{< ref "docs/deployment/elastic_scaling" >}}#adaptive-batch-scheduler) supports speculative execution.
+- `jobmanager.adaptive-batch-scheduler.speculative.enabled: true`
+
+### Tuning Configuration
+To make speculative execution work better for different jobs, you can tune below configuration options of the scheduler:
+- [`jobmanager.adaptive-batch-scheduler.speculative.max-concurrent-executions`]({{< ref "docs/deployment/config" >}}#jobmanager-adaptive-batch-scheduler-speculative-max-concurrent-e): 
+Controls the maximum number of execution attempts of each operator that can execute concurrently, including the original one and speculative ones.
+- [`jobmanager.adaptive-batch-scheduler.speculative.block-slow-node-duration`]({{< ref "docs/deployment/config" >}}#jobmanager-adaptive-batch-scheduler-speculative-block-slow-node): 
+Controls how long an detected slow node should be blocked for.
+
+You can also need to tune below configuration options to control the slow task detector:
+- [`slow-task-detector.check-interval`]({{< ref "docs/deployment/config" >}}#slow-task-detector-check-interval): 
+The interval to check slow tasks.
+- [`slow-task-detector.execution-time.baseline-lower-bound`]({{< ref "docs/deployment/config" >}}#slow-task-detector-execution-time-baseline-lower-bound): 
+The lower bound of slow task detection baseline.
+- [`slow-task-detector.execution-time.baseline-multiplier`]({{< ref "docs/deployment/config" >}}#slow-task-detector-execution-time-baseline-multiplier): 
+The multiplier to calculate the slow tasks detection baseline. Given that the parallelism is N and the ratio is R, define T as the median of the first N*R finished tasks' execution time. The baseline will be T*M, where M is the multiplier of the baseline.
```

Comment:
N*R should be quoted with `, otherwise the * will be recognized as italic symbol

Reply:
Same as T*M